[exporter/prometheusexporter] Update wording in README (#10201)
open-telemetry/opentelemetry-collector-contrib: exporter/prometheusexporter/README.md

# Prometheus Exporter

> Exports data in a [Prometheus format](https://prometheus.io/docs/concepts/data_model/). Allows scraping them by the [Prometheus](https://prometheus.io/) back-end.

Reviewer: ```suggestion
Exports data in the [Prometheus format](https://prometheus.io/docs/concepts/data_model/), which allows it to be scraped by a [Prometheus](https://prometheus.io/) server.
```

Supported pipeline types: metrics

## Getting Started

The following settings are required:

> - `endpoint` (no default): the address on which data will be exposed and against which the Prometheus scrape handler will be run on.

Reviewer: ```suggestion
- `endpoint` (no default): the address on which metrics will be exposed by the Prometheus scrape handler.
```
"prometheus scrape handler" is a reference to the http "handler" in this exporter, rather than referring to the prom server.

Author: Ah, makes sense, thanks!
